app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedListAdapter.kt

```diff
@@ -194,8 +194,8 @@ class OngoingListAdapter(
   private class OngoingStoryViewHolder(
```

Comment:
Rename `OngoingStoryViewHolder ` to `PromotedStoryViewHolder`

Reply:
done